Epithelial.cpp

```diff
@@ -0,0 +1,88 @@
+/////////////////////////////////////////////////////////////////////////////////////////
+//
+// gro is protected by the UW OPEN SOURCE LICENSE, which is summarized here.
```

Comment:
Since you are not at UW, you need to change the licensing information here to be for Caltech.  I suggest using a BSD 3-clause license.  The wording is similar, but slightly different. You can find an example [here](https://github.com/python-control/python-control/blob/master/LICENSE).